src/interface/bme280_sensor.py

```diff
@@ -16,15 +16,15 @@ def __init__(self, name, addr, i2c_helper):
     def readData(self):
         self.data = bme280.sample(self.i2c_helper.i2c, self.address)
 
-    @Reading(units='°C')
+    @Reading(units=' °C')
```

Comment:
Why are you modifying these 'units' strings?

Reply:
Yes, locally it's correct to not have a space, as it was before this commit. However, I would welcome the addition of localization to sensor units.

Reply:
Placing a space between text and units can enhance recognition